src/bioregistry/data/bioregistry.json

```diff
@@ -31841,6 +31841,39 @@
       }
     ]
   },
+  "dotad": {
+    "contact": {
+      "email": "hj@uestc.edu.cn",
+      "name": "Jian Huang",
+      "orcid": "0000-0003-2314-7104"
+    },
+    "contributor": {
+      "email": "ji.kang@northeastern.edu",
+      "github": "kanghosaeyo",
+      "name": "Kangho Ji",
+      "orcid": "0009-0009-3402-8013"
+    },
+    "description": "Identifiers represent therapeutic antibody entries with developability assessments in the Database Of Therapeutic Antibody Developability (DOTAD), which curates antibody sequence data and developability metrics including aggregation propensity, immunogenicity risk, stability, and manufacturability to support computational modeling and therapeutic antibody development.",
+    "github_request_issue": 1730,
+    "homepage": "https://i.uestc.edu.cn/DOTAD",
+    "keywords": [
+      "antibody development",
+      "developability",
+      "therapeutic antibody"
+    ],
+    "name": "Database of Therapeutic Antibody Developability",
+    "example": "alirocumab",
+    "pattern": "^[a-z]+$",
+    "publications": [
+      {
+        "doi": "10.1007/s12539-024-00613-2",
+        "pubmed": "38530613",
+        "title": "DOTAD: A Database of Therapeutic Antibody Developability",
+        "year": 2024
+      }
+    ],
+    "uri_format": "https://i.uestc.edu.cn/DOTAD/detail.html?INN=$1"
```

Comment:
Could this just be a search for international nonproprietary names (INNs)? Those are string literals and I don’t think qualify to get registered as an identifier space.